runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -563,24 +594,28 @@ public void onGraphicChanged(GraphicChanged event)
 				}
 			}
 
+			// threshold to allow frozen graphic to register and remove redundant timers
 			if (actor.getGraphic() == ICERUSH.getGraphicId())
 			{
-				createGameTimer(ICERUSH);
+                removeGameTimer(ICEBARRAGE, 1);
+                createGameTimer(ICERUSH, false);
 			}
 
 			if (actor.getGraphic() == ICEBURST.getGraphicId())
 			{
-				createGameTimer(ICEBURST);
+                removeGameTimer(ICEBARRAGE, 1);
+                createGameTimer(ICEBURST, false);
 			}
 
 			if (actor.getGraphic() == ICEBLITZ.getGraphicId())
 			{
-				createGameTimer(ICEBLITZ);
+                removeGameTimer(ICEBARRAGE, 1);
+                createGameTimer(ICEBLITZ, false);
 			}
 
 			if (actor.getGraphic() == ICEBARRAGE.getGraphicId())
 			{
-				createGameTimer(ICEBARRAGE);
+				;
```

Comment:
? remove this condition now I guess, if it is not necessary

Reply:
Sure, wanted to be explicit as other people might look at the code